pages/docs/adapters/calculix/adapter-calculix-get-adapter.md

```diff
@@ -12,14 +12,15 @@ After [installing preCICE](installation-overview.html) and [getting the CalculiX
 1. Download and unzip the adapter (e.g. in the `CaluliX` folder):
 
     ```bash
-    wget https://github.com/precice/calculix-adapter/archive/master.zip 
-    unzip master.zip 
-    cd calculix-adapter-master
+    wget https://github.com/precice/calculix-adapter/archive/refs/heads/v{{site.calculix_version}}.zip 
+    unzip v{{site.calculix_version}}.zip 
+    cd calculix-adapter-{{site.calculix_version}}
```

Comment:
Unrelated to the PR, but in the long run, would'nt it be better to point to master (while writing in the doc that master is version 2.XX) instead of the branch stored in the website ?
The game then would be to ensure the website and master stay syncrhonized.

But this is just another instance of the debate of how we handle different versions I guess.

Reply:
Actually, we should make adapter releases and point to them.

Reply:
Let's update this in a separate PR, after we have the first release.